execute/factory.go

```diff
@@ -47,10 +47,12 @@ func (p PluginFactory) NewReportingPlugin(
 	config ocr3types.ReportingPluginConfig,
 ) (ocr3types.ReportingPlugin[[]byte], ocr3types.ReportingPluginInfo, error) {
 	return NewPlugin(
-		context.Background(),
 		config,
 		cciptypes.ExecutePluginConfig{},
 		nil,
+		nil,
+		nil,
+		nil,
 	), ocr3types.ReportingPluginInfo{}, nil
```

Comment:
I think you should at least specify the max report size here in the `Limits` field (declared in plugin.go as a const)

Reply:
Is this something in the HomeChain? I guess it could vary per destination.

Reply:
Maybe they'd vary by family for the reports, outcomes are fully off-chain though so you could set them here family-agnostically